apps/console/src/features/applications/components/forms/advanced-configurations-form.tsx

```diff
@@ -361,143 +362,181 @@ export const AdvancedConfigurationsForm: FunctionComponent<AdvancedConfiguration
+                                        <Grid.Row
+                                            columns={ 1 }
+                                            onClick={ handleWhenClientAttestationClickedWhenDisabled }>
+                                            <Grid.Column >
+                                                <Heading as="h5" bold="500">
+                                                    <GenericIcon
+                                                        size="micro"
+                                                        icon={ getTechnologyLogos().android }
+                                                        verticalAlign="middle"
+                                                        floated="left"
+                                                    />
+                                                    { t("console:develop.features.applications.forms.advancedConfig." +
+                                                        "sections.applicationNativeAuthentication.fields." +
+                                                "android.heading") }
+                                                </Heading>
+                                                <Field.Input
+                                                    ariaLabel="Android package name"
+                                                    inputType="default"
+                                                    name="androidPackageName"
+                                                    label={ t("console:develop.features.applications.forms." +
+                                                    "advancedConfig." +
+                                                    "sections.applicationNativeAuthentication.fields." +
+                                                    "android.fields.androidPackageName.label") }
+                                                    required={ false }
+                                                    readOnly={ readOnly }
+                                                    value={ config?.attestationMetaData?.androidPackageName ?
+                                                        config?.attestationMetaData?.androidPackageName : "" }
+                                                    placeholder={ t("console:develop.features.applications.forms." +
+                                                    "advancedConfig." +
+                                                    "sections.applicationNativeAuthentication.fields." +
+                                                    "android.fields.androidPackageName.placeholder") }
+                                                    hint={ t("console:develop.features.applications.forms." +
+                                                    "advancedConfig." +
+                                                    "sections.applicationNativeAuthentication.fields." +
+                                                    "android.fields.androidPackageName.hint") }
```

Comment:
Better to keep an indentation here.
```suggestion
                                                    hint={ t("console:develop.features.applications.forms." +
                                                        "advancedConfig." +
                                                        "sections.applicationNativeAuthentication.fields." +
                                                        "android.fields.androidPackageName.hint") }
```